package.json

```diff
@@ -31,7 +31,9 @@
     "format": "prettier --write \"**/*.{js,ts}\"",
     "format:check": "prettier --list-different \"**/*.{js,ts}\"",
     "lint": "tslint --project tsconfig.json --fix",
-    "lint:check": "tslint --project tsconfig.json"
+    "lint:check": "tslint --project tsconfig.json",
+    "docs:2": "mkdir -p docs/1-Guides && mkdir -p docs/2-API && cp documentation/*.md docs/1-Guides && api-extractor run --local && generate-ts-docs markdown -i docs/2-API -o docs/2-API && cp README.md docs/ && docsify-auto-sidebar",
```

Comment:
Rename: replace the script docs of today by yours (and remove typedoc from the package.json). Your generated documentation looks clearly better ;)

Reply:
As the new script relies on the *.d.ts files, you should edit the https://github.com/dubzzz/fast-check/blob/master/.travis.yml#L63:L76 script and add `npm run build:publish-types` before the existing `npm run docs` (and remove `mkdir` from the travis step as you already do that in the script itself).